src/main/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReads.scala

```diff
@@ -108,6 +109,12 @@ class CallDuplexConsensusReads
  val maxReadsPerStrand: Option[Int] = None,
  @arg(doc="The number of threads to use while consensus calling.") val threads: Int = 1,
  @arg(doc="Consensus call overlapping bases in mapped paired end reads") val consensusCallOverlappingBases: Boolean = true,
+ @arg(doc="""
+            |Pull reads from this many source molecules into memory for multi-threaded processing.
+            |Using a smaller value will require less memory but will negatively impact processing speed.
+            |For very large family sizes, a smaller value may be necessary to reduce memory usage.
+            |This value is only used when `--threads > 1`.
+          """) val maxSourceMoleculesInMemory: Int = DefaultChunkSize
```

Comment:
Do we recommend this be a multiple of threads for when threads > 1? I'm imagining that would be the most efficient guidance if family sizes and the computation time to create a consensus was stable.

Reply:
I don't necessarily think it is going to make a difference https://github.com/fulcrumgenomics/fgbio/pull/867#discussion_r935847745